plugins/plugin-chart-echarts/src/constants.ts

```diff
@@ -53,3 +53,9 @@ export const LABEL_POSITION: [LabelPositionEnum, string][] = [
   [LabelPositionEnum.InsideTopRight, 'Inside top right'],
   [LabelPositionEnum.InsideBottomRight, 'Inside bottom right'],
 ];
+
+export const OPACITY = {
```

Comment:
The enumerated type follows the PascalCase style.

```suggestion
export enum OpacityEnum {
  Transparent: 0,
  SemiTransparent: 0.3,
  NonTransparent: 1,
}
```

Reply:
If we only have `Transparent`, `SemiTransparent` and `NonTransparent`, then the constants make sense. If there are many enumerations, it makes no sense. 

Do we have many transparency rate use cases now?

Reply:
@xiezhongfu please help review 🙏

Reply:
In the gauge chart, I use `0.1` as the transparency value, which looks more suitable.

Reply:
The purpose of the design here is to standardize the constant values of the `opacity rate`, rather than reset them for each viz and different name them. 
- the `COLOR_ALPHA` used in Treemap for the `semi-transparent`
- All except gauge chart are used 0.3 for the `semi-transparent` 

I suggest that this semi-transparent should be consistent in all viz.

![image](https://user-images.githubusercontent.com/2016594/124872461-e19cd500-dff7-11eb-915b-e6269b4dead1.png)

Reply:
You are right！Thanks for the explanation.I will use this constant in all charts.

Reply:
Thanks a lot!